flink-core/src/test/java/org/apache/flink/api/common/typeutils/TypeSerializerUpgradeTestBase.java

```diff
@@ -50,7 +50,7 @@
  */
 public abstract class TypeSerializerUpgradeTestBase<PreviousElementT, UpgradedElementT> extends TestLogger {
 
-	protected static final MigrationVersion[] migrationVersions = new MigrationVersion[]{
+	public static final MigrationVersion[] migrationVersions = new MigrationVersion[]{
```

Comment:
This changed from `protected` to `public` because we want to reference `migrationVersions` from `ScalaCaseClassSerializerUpgradeTest`, I'm not sure this is the best solution